src/jit/importer.cpp

```diff
@@ -1146,8 +1146,7 @@ GenTree* Compiler::impAssignStructPtr(GenTree*             destAddr,
     assert(src->gtOper == GT_LCL_VAR || src->gtOper == GT_FIELD || src->gtOper == GT_IND || src->gtOper == GT_OBJ ||
```

Comment:
Maybe it would be nice to replace this long list of `gtOper == one || gtOper == two || ,,,` with `src->OperIs(one, two, ...)`.

Reply:
Good suggestion